### ISSUE TITLE
Some records in bodyfile.txt have the wrong format

### ISSUE THREAD

**Issue by mnrkbys** [tclahr/uac#267]
I am checking the bodyfile.txt which is generated by the development version of UAC.
After generating the bodyfile.txt, I tried to generate the timeline with TSK's mactime.
However, the following error messages were displayed.

```
$ mactime -z UTC -d -b bodyfile/bodyfile.txt 
Argument "16878293rt_Global_Root_G2.pem -> /usr/share/ca-certifica..." isn't numeric in sprintf at /usr/bin/mactime line 397, <BODY> line 493.
Argument "168782n/Downloads/src/bomutils/.git/config" isn't numeric in sprintf at /usr/bin/mactime line 397, <BODY> line 3255.
Argument "1721978kup/lib/python3.10/site-packages/pip/_vendor/rich..." isn't numeric in sprintf at /usr/bin/mactime line 398, <BODY> line 28766.
Argument "17219782cpython-310.pyc" isn't numeric in sprintf at /usr/bin/mactime line 397, <BODY> line 28831.
Argument "172l5lib/__pycache__/__init__.cpython-310.pyc" isn't numeric in sprintf at /usr/bin/mactime line 398, <BODY> line 29144.
Argument "168783147-r--" isn't numeric in sprintf at /usr/bin/mactime line 397, <BODY> line 29414.
Argument "/home/john/envs/munin/lib/python3.10/site-packages/pip/_..." isn't numeric in sprintf at /usr/bin/mactime line 397, <BODY> line 32480.
Argument "1.cpython-310.pyc" isn't numeric in sprintf at /usr/bin/mactime line 398, <BODY> line 32646.
Argument "1721976w-r--" isn't numeric in numeric lt (<) at /usr/bin/mactime line 388, <BODY> line 32936.
Argument "172197626_/align.cpython-310.pyc" isn't numeric in sprintf at /usr/bin/mactime line 398, <BODY> line 33207.
Argument "1721976277rw-r--" isn't numeric in sprintf at /usr/bin/mactime line 396, <BODY> line 33280.
(snip)
```

For example, the line 493 of the bodyfile.txt looks like below:

```
0|/home/john/Downloads/src/volatility3/volatility3/framework/plugins/linux/psscan.py|1573343|-rw-rw-r--|1000|1000|6722|1687831692|1687829320|16878293rt_Global_Root_G2.pem -> /usr/share/ca-certificates/mozilla/DigiCert_Global_Root_G2.crtRoot_CA.crtG2.crtRootCA_2015.crt.crt|2361711|lrwxrwxrwx|0|0|62|1722838897|1687828690|1687828690|1687828690
```

The above record appears to be a mixture of two records.

**Comment by tclahr** [comment]
This is due to xargs max-procs parameter. I will remove it and submit a new PR.

**Comment by tclahr** [comment]
Fixed in PR #269
